pulsar-io/kinesis/pom.xml

```diff
@@ -83,36 +83,12 @@
 	<!-- /kinesis dependencies -->
 
   </dependencies>
-  
+
   <build>
     <plugins>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
pulsar function requires to submit uber jar so, it loads all required classes for the function. Therefore, it's necessary to create a shaded jar here? with this change, it's no more uber jar. now, user has to explicitly create a module to create shaded jar which I think we should avoid here.??

Reply:
how are we planning to unpack nar and load dependent jars in function framework?